### PR TITLE
(PC-4418) offer description level 1

### PR DESCRIPTION
**Pull request by Giregm**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-4418

## Checklist

I have:

- [x] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [x] Made sure my feature is working on the relevant real / virtual devices.
- [x] Written **unit tests** for my feature.
- [x] Added new reusable components to the AppComponents page and communicate to the team on slack
- [x] Added a **screenshot** for UI tickets.

## Screenshots

| \*iOS - [Phone name] | \*iOS - [Phone name] | Android - [Phone name] | Android - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
|         image|           image               |            image        |             image         |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn version:testing` (this will create a commit with a tag)
- then run `git push --follow-tags`

### CODE REVIEW

File: src/features/offer/components/__tests__/OfferPartialDescription.test.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { render } from '@testing-library/react-native'
+import React from 'react'
+
+import { OfferPartialDescription } from '../OfferPartialDescription'
+
+const description =
+  'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua: https://google.com'
+
+describe('OfferPartialDescription', () => {
+  it('centers CTA when provided description is falsy', () => {
```
Comment (DanielBaudry) [inline src/features/offer/components/__tests__/OfferPartialDescription.test.tsx:10]:
```suggestion
  it('centers CTA when provided description is empty', () => {
```

**Review by blixit — APPROVED** [review]

File: src/locales/fr/messages.po [hunk 2]
```diff
@@ -363,6 +367,10 @@ msgstr "s'appliquent."
 msgid "tonadresse@email.com"
 msgstr "tonadresse@email.com"
 
+#: src/features/offer/atoms/OfferSeeMore.tsx:10
+msgid "voir plus"
+msgstr "voir plus"
```
Comment (DanielBaudry) [inline src/locales/fr/messages.po:372]: Il manque peut-être une majuscule ?

**Review by DanielBaudry — APPROVED** [review]

File: src/locales/fr/messages.po (hunk 2 above)
Comment (Giregm) [reply src/locales/fr/messages.po:372]: image

Comme ça suit un texte souvent tronqué ça ne me choque pas :/

File: src/features/offer/components/__tests__/OfferPartialDescription.test.tsx (hunk 1 above)
Comment (Giregm) [reply src/features/offer/components/__tests__/OfferPartialDescription.test.tsx:10]: Merci! 
Je me suis permit de rebase ça dans le 2nd commit

**Pull request merged**